src/like-service/app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -12,6 +12,6 @@ class VerifyCsrfToken extends Middleware
      * @var array
      */
     protected $except = [
-        //
+        '/like-service/like-post'
```

Comment:
[DISCUSS] I think we should just remove this middleware

Reply:
Discussed. Removing would cause further issues